pkg/ioutils/buffer_test.go

```diff
@@ -20,7 +20,7 @@ func TestFixedBufferWrite(t *testing.T) {
 		t.Fatalf("expected \"hello\", got %q", string(buf.buf[:5]))
 	}
 
-	n, err = buf.Write(bytes.Repeat([]byte{1}, 64))
+	_, err = buf.Write(bytes.Repeat([]byte{1}, 64))
```

Comment:
`n` should be checked here to have the partial write before `errBufferFull`, allowing the caller to recover the amount written.